drivers/dma/dma_dw_common.c

```diff
@@ -206,6 +206,7 @@ int dw_dma_config(const struct device *dev, uint32_t channel,
 		case 1:
 			/* byte at a time transfer */
 			lli_desc->ctrl_lo |= DW_CTLL_SRC_WIDTH(0);
+			break;
```

Comment:
This seems like a legitimate bug and could be sent outside of this PR, would be approved quickly.

Reply:
Yeah, though of this but I felt this might not take long. If it is taking long, I will push it through a new PR.